node/src/components/consensus/era_supervisor.rs

```diff
@@ -418,16 +421,19 @@ where
     pub(super) fn handle_message(&mut self, sender: I, msg: ConsensusMessage) -> Effects<Event<I>> {
         match msg {
             ConsensusMessage::Protocol { era_id, payload } => {
-                if era_id.0 + BONDED_ERAS < self.era_supervisor.current_era.0 {
+                if era_id.0 + 2 * BONDED_ERAS < self.era_supervisor.current_era.0 {
```

Comment:
I will be grateful if you extract it to a method and give it a comment.

Reply:
This can actually be removed entirely, because it's already handled in `delegate_to_era` anyway.